catalogue_pipeline/id_minter/src/test/scala/uk/ac/wellcome/platform/idminter/steps/IdEmbedderTests.scala

```diff
@@ -95,8 +95,8 @@ class IdEmbedderTests
         )
     ).thenReturn(Try(throw expectedException))
 
-    val newWorkFuture = idEmbedder.embedId(
-      json = parse(JsonUtil.toJson(originalWork).get).right.get)
+    val newWorkFuture =
+      idEmbedder.embedId(json = parse(toJson(originalWork).get).right.get)
```

Comment:
Sending a work to a JSON string, then unpacking it as a Circe Json object, seems moderately odd.

Maybe we need `toJsonString` and `toCirceJson` in the utils file (or similar)? Depends on how frequently it comes up elsewhere.